src/module.ts

```diff
@@ -69,9 +72,25 @@ export default defineNuxtModule<ModuleOptions>({
       maxAge: 0,
     })
 
+    const enableSSR = !nuxt.options.dev
+      && nuxt.options.ssr
+      && !nuxt.options._generate
+      && !!options.enableSSR
+      && options.strategies !== 'injectManifest'
+
+    const ssrPages: NuxtPage[] = []
+    let nitroOptions: NitroOptions | undefined
     nuxt.hook('nitro:init', (nitro) => {
```

Comment:
maybe we can just remove this and configure the options in `vite:extendConfig`  hook